src/Compilers/CSharp/Portable/Binder/Binder_Expressions.cs

```diff
@@ -4830,6 +4835,15 @@ private bool IsUsingAliasInScope(string name)
                 return BadExpression(node, boundLeft);
             }
 
+            // PROTOTYPE(default) unify with case above
+            // No member accesses on default
```

Comment:
I looked into this and it was thorny.
What I tried was to have `MakeMemberAccessValue` return a `BadExpression` and report an error for those two cases. But it made the diagnostics significantly worse in a number of cases (especially lambda). I'll stop by to discuss tomorrow.